jobs/env/ci-kubernetes-e2e-scalability-common.env

```diff
@@ -14,7 +14,9 @@ KUBEPROXY_TEST_ARGS=--profiling
 SCHEDULER_TEST_ARGS=--profiling
 # Switch off image puller to workaround #44701
 PREPULL_E2E_IMAGES=false
-# Increase resync period to simulate production
+# Reduce logs verbosity.
+TEST_CLUSTER_LOG_LEVEL=--v=2
```

Comment:
Sanity check - are we overriding this to v4 for some jobs? (I remember we were doing for some small scale jobs to have more logs)

Reply:
Also will this override v1 with v2 for 5k jobs?

Reply:
only for kubemarks.
For real clusters (which this is above), we already had --v=2 for 100-nodes and 2000-nodes. and --v=1 for 5k-nodes (that that still is in the config).

It will no override it, because the 5k-node specific one is referenced later so this one actually overwrites this one (I'm planning to remove that too, but not now).

Reply:
Ok thanks, just wanted to confirm.